opencog/atoms/base/Atom.h

```diff
@@ -190,6 +195,16 @@ class Atom
         return classserver().isA(at, t);
     }
 
+    /** Returns the outgoing set.
+     *
+     * @return The outgoing set.
+     */
+    virtual inline const HandleSeq& getOutgoingSet() const
+    {
+        static HandleSeq no_atoms;
+        return no_atoms;
+    }
```

Comment:
Oh, I see .. the old code that this replaced did not throw.  However .. we probably should. Do the unit tests pass if we throw?